ReactAndroid/src/main/java/com/facebook/react/modules/statusbar/StatusBarModule.java

```diff
@@ -184,7 +185,28 @@ public void setStyle(@Nullable final String style) {
       return;
     }
 
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+      UiThreadUtil.runOnUiThread(
+        new Runnable() {
+          @TargetApi(Build.VERSION_CODES.R)
+          @Override
+          public void run() {
+            WindowInsetsController insetsController = activity.getWindow().getInsetsController();
+            if ("light-content".equals(style)) {
+              // light-content means white icons on a dark status bar
+              insetsController.setSystemBarsAppearance(
+                0,
```

Comment:
Documentation says that this is equivalent to `WindowInsetsController.BEHAVIOR_SHOW_BARS_BY_TOUCH`. Was this intentional?

Reply:
The second parameter is `mask`, so it should only reset `APPEARANCE_LIGHT_STATUS_BARS`, I believe 🙂

Reply:
Correct, the second parameter is a mask and it's the recommended way to clear this flag per https://developer.android.com/reference/kotlin/android/view/WindowInsetsController#setsystembarsappearance

Reply:
Yup you're right. The method docs actually say that this is the correct way to clear the flags 👍